pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -493,44 +493,27 @@ func manageOpenShiftControllerManagerDeployment_v311_00_to_latest(
 		}
 	} else {
 		for i, c := range required.Spec.Template.Spec.Containers {
```

Comment:
Setting the values should be tested by https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/0ab30996207ada63ef2e2fd4f47aac03b0872c9a/pkg/operator/sync_openshiftcontrollermanager_v311_00_test.go#L780

Can we improve the test so it tests also the update part? Should be enough to just update the proxy config and call `manageOpenShiftControllerManagerDeployment_v311_00_to_latest` again.

Reply:
Amended.